gradle.properties

```diff
@@ -17,16 +17,9 @@ org.gradle.jvmargs=-Xmx4096m -Dfile.encoding=UTF-8
 android.useAndroidX=true
 # Automatically convert third-party libraries to use AndroidX
 android.enableJetifier=true
-# Namespace each library's R class
-# https://developer.android.com/studio/releases/gradle-plugin#4.1-nontransitive-r-class
-android.nonTransitiveRClass=true
 # Kotlin code style for this project: "official" or "obsolete":
 kotlin.code.style=official
 # Xcode project path for the KMM plugin
 xcodeproj=./ios
 # We're aware that MP is an alpha feature
 kotlin.mpp.stability.nowarn=true
-# Enable the commonizer
-# https://kotlinlang.org/docs/reference/migrating-multiplatform-project-to-14.html#try-the-hierarchical-project-structure
-kotlin.mpp.enableGranularSourceSetsMetadata=true
-kotlin.native.enableDependencyPropagation=false
```

Comment:
These flags are still necessary as per the linked documentation

Reply:
got it